### PR TITLE
WIP: Bail when webpack build errors.

### LINKED ISSUE #2895: webpack production build should fail on warning and errors
webpack's production build should fail (and stop image building) on warnings and errors to prevent issues like the one fixed in #2894.

### PR DESCRIPTION
**Pull request by jezdez**
Fixes #2895.

### CODE REVIEW

File: client/app/components/QueryEditor.jsx [hunk 1]
```diff
@@ -208,7 +208,7 @@ class QueryEditor extends React.Component {
                 {this.props.dataSources.map(ds => <option label={ds.name} value={ds.id} key={`ds-option-${ds.id}`}>{ds.name}</option>)}
               </select>
               {this.props.canEdit ?
-                <Tooltip placement="top" title={modKey + ' + S'}>
+                <Tooltip placement="top" title={modKey + ' + S'>}>
```
Comment (kravets-levko) [inline client/app/components/QueryEditor.jsx]: Seems it's invalid jsx (or am I wrong?)

Comment (arikfr) [reply client/app/components/QueryEditor.jsx]: It's probably here to test that the build actually fails :)

Comment (kravets-levko) [reply client/app/components/QueryEditor.jsx]: Probably :-) But I think we shouldn't keep it :-)

Comment (jezdez) [reply client/app/components/QueryEditor.jsx]: Ooops, yeah, so this was hard to get right and I wanted to have a real-life example to test this with. Will remove once I have this working as expected.

**Pull request closed without merging**